proposals/2024-06-14_mixxx_e_v.md

```diff
@@ -0,0 +1,121 @@
+## Your Proposal Title
+
+* **Owners:**
+  * Daniel Schürmann
+
+* **Implementation Status:** `Not implemented`
+
+* **Related Issues and PRs:**
+
+* **Other docs or links:**
+  * https://ev.kde.org/corporate/statutes/
+  * https://phabricator.kde.org/T12954
+
+* This design doc shall be used to file our Articles of Assoziation 
+
+## Draft
+
+# Articles of Association - Mixxx e.V.
+
+As a non-profit corporation incorporated under German law, Mixxx e.V. is required to declare its intentions and by-laws in a formal "Articles of Association" document. The original and legally binding version is the German version. The following translation is for information only.
+
+## Clarification of Terminology
+
+1. Computer programs considered as free software or "open source software" are put into the public domain free of charge by the originator, allowing others the freedom to use, study, adapt, copy, and improve the program for the common good.
+2. Developing free software involves research, compilation of principles and concepts, and their testing through programming and testing of free software.
+3. "Mixxx" is a DJ software developed as free software and available to the public free of charge.
```

Comment:
We should elaborate "DJ software" here, until now nowhere is mentioned, that Mixxx has something to do with DJing/music